tests/testtlswebsockets.nim

```diff
@@ -149,7 +149,6 @@ suite "Test websocket TLS transmission":
         check request.uri.path == "/wss"
         let ws = await createServer(request, "proto")
         await ws.send(testString)
-        await ws.stream.closeWait()
```

Comment:
Hmm, actually we should not call `closeWait()` on the stream - it should be private. We should only call `ws.close()`.